src/components/attendee/SplashGated/SplashSpace.hoc.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
there's an identical file: src/components/attendee/SplashSpace/SplashSpace.hoc.tsx
do we need both?

Reply:
Nice catch! Removed in https://github.com/sparkletown/sparkle/pull/3012/commits/2a4fe1dc4ef15783ba8a9a8fe44efb9b96c19540